src/hooks/useMeta.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// adapted from
```

Comment:
Isn't this comment related to CardErrorBoundary ?

Reply:
Removed